data/phi0690/phi003/phi0690.phi003.perseus-lat2.xml

```diff
@@ -36,18 +35,15 @@
                 <biblStruct>
                     <monogr>
                         <author>Vergil</author>
-                        <author>Virgil</author>
                         <title>The Bucolics, Aeneid, and Georgics Of Virgil</title>
                         <editor role="editor">J. B. Greenough</editor>
```

Comment:
`<editor role="editor">` s/b `<editor>`

Reply:
I'm not quite sure what you mean by s/b @lcerrato sorry if I am missing the obvious.

Reply:
Sorry, should be

Reply:
So its ok that I took out the role="editor" right that's what I have been doing.